Rem/Bot/DiscordBot.cs

```diff
@@ -47,33 +45,39 @@ private static Task Log(string msg)
 
         private async Task InstallCommands()
         {
-            await _dbContext.CreateTableAsync<Models.Image>();
+            await _dbContext.CreateTableAsync<Image>();
 
             _services.AddSingleton(_dbContext);
             _services.AddSingleton(_state);
-            
+
             _client.MessageReceived += MessageReceived;
             await _commands.AddModulesAsync(Assembly.GetEntryAssembly());
         }
 
         private async Task MessageReceived(SocketMessage messageParam)
         {
             // Don't process the command if it was a System Message
-            var message = messageParam as SocketUserMessage;
-            if (message == null) return;
+            if (!(messageParam is SocketUserMessage message)) return;
```

Comment:
I use "as" instead of "is" because using "is" means that we require an extra cast if we want to use attributes/methods of SocketUserMessage.